pkgs/development/tools/protoc-gen-prost-crate/default.nix

```
@@ -0,0 +1,28 @@
+{ fetchCrate
+, lib
+, rustPlatform
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "protoc-gen-prost-crate";
+  version = "0.3.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-MtGeU2PnVYPXb3nly2UaryjmjMz1lxcvYDjFiwf58FA=";
+  };
+
+  cargoSha256 = "sha256-dcKJRX/iHIWEmBD2nTMyQozxld8b7dhxxB85quPUysg=";
+
+  cargoBuildFlags = ["--bin" pname];
+
+  nativeBuildInputs = [ protobuf ];
+
+  meta = with lib; {
+    description = "Protocol Buffers compiler plugin powered by Prost!";
+    homepage = "https://github.com/neoeinstein/protoc-gen-prost";
```

```suggestion
    homepage = "https://github.com/neoeinstein/protoc-gen-prost";
    changelog = "https://github.com/neoeinstein/protoc-gen-prost/blob/main/CHANGELOG.md";
```

and for the other packages as well.